docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -0,0 +1,585 @@
+:linkTroubleshooting: https://github.com/integr8ly/example-customisations/blob/master/docs/troubleshooting.adoc
+:linkGitHubFork: https://help.github.com/articles/fork-a-repo/
+:linkIndexURL: https://github.com/integr8ly/example-customisations/index.adoc
+:walkthrough-1A: *Integrating message-oriented middleware with a RESTful API using AMQ Online*
+:walkthrough-2: *Managing and protecting APIs using API management*
+:publishing-walkthrough: *Adding walkthroughs to a cluster*
+
+= Writing {PRODUCT} Walkthroughs
```

Comment:
This topic is more than a "how to" I would just call it {Product Walkthroughs}

Reply:
The only info that I want to put in here is about 'Writing Walkthroughs', so I'm sure your comment is correct, but I'd prefer to address it by removing or explaining any specious content